legal-api/migrations/versions/3cad4247b576_create_shadow_business_shadow_filing_.py

```diff
@@ -0,0 +1,57 @@
+"""create shadow_business, shadow_filing, and legacy_outputs tables
+
+Revision ID: 3cad4247b576
+Revises: 08da65c4a94a
+Create Date: 2023-03-30 16:04:24.255002
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '3cad4247b576'
+down_revision = '08da65c4a94a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table('shadow_filings', 
+        sa.Column('id', sa.Integer(), autoincrement=False, nullable=False),
+        sa.Column('business_id', sa.Integer(), autoincrement=False, nullable=True),
+        sa.Column('submitter_id', sa.Integer(), autoincrement=False, nullable=True),
+        sa.Column('effective_date', sa.DateTime(timezone=True), autoincrement=False, nullable=True),
+        sa.Column('has_legacy_outputs', sa.Boolean(), nullable=True),
+        sa.Column('temp_reg', sa.String(10), nullable=True),
+        sa.Column('colin_event_id', sa.Integer(), autoincrement=False, nullable=True),
+        sa.ForeignKeyConstraint(['business_id'], ['businesses.id'], ),
+        sa.ForeignKeyConstraint(['temp_reg'], ['registration_bootstrap.identifier'],),
+        sa.ForeignKeyConstraint(['submitter_id'], ['users.id']),
+        sa.ForeignKeyConstraint(['colin_event_id'], ['colin_event_id.colin_event_id']),
+        sa.PrimaryKeyConstraint('id')
+    )
+
+    op.create_table('shadow_business', 
```

Comment:
i'm not entirely sure of all the fields you will need for this shadow business table but `legal_type`, `identifier`, `state` will probably useful.